modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/ballooneditor/plugin.js

```diff
@@ -369,9 +369,11 @@
 			CKEDITOR.ui.panel.prototype.showBlock = function (name) {
 				if (!this.name) {
 					this.name = name;
+
+					return originalShowBlockFn.call(this, this.name);
```

Comment:
Non-blocking - we can iterate on this in https://issues.liferay.com/browse/LPS-136119

The logic seems odd to me, both in the old and the new version. The purpose of the is original function was apply new `name` only if old one does not exist. In the new one, we are applying `name` always, but setting `this.name` only if it's falsy. Why not update `this.name` always?

Reply:
I applied this code because the received `name` may be different from the `this.name`. When debugging panel things it was an issue however it breaks the bgcolor thing. When calling `name` instead of `this.name` the buttonBlock can find the respective value of the panel, at least in the first mounting. 

> Why not update this.name always?

Because `this.name` may contain an invalid value.

Reply:
I still smells funny to me, but let's table this for now, returning to it in https://issues.liferay.com/browse/LPS-136119